src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-ref-architecture.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Reference architecture for OpenTelemetry with New Relic
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: Use this OpenTelemetry reference architecture to discuss implementation with your team.
+---
+If you need guidance implementing OpenTelemetry with New Relic One, this reference architecture should help your software developers, DevOps, enterprise architects, and technology managers collaborate on the project. It is a template showing a bird's-eye view of common ways to implement OpenTelemetry with New Relic.
+
+The diagram shows a New Relic One implementation that ingests telemetry from OpenTelemetry, Prometheus, Jaeger, and a New Relic APM agent. The three services (A, B, and C) are instrumented to send metrics, traces, and logs.
```

Comment:
the 'a New Relic One implementation' struck me as slightly awkward. New Relic, or New Relic One, is just part of the implementation. I'm not sure it makes sense to call it a 'New Relic One implementation', maybe needs a more general wording, maybe just 'The diagram shows an implementation where..."

Reply:
@zuluecho9 that's a great suggestion! I'll use your simplified version!